src/hooks/context/useResetContext.js

```diff
@@ -9,7 +9,7 @@ import TokenStorage from '../../utils/storage/custom/tokenStorage';
 import TopicPageStorage from '../../utils/storage/custom/topicPageStorage';
 import UserSchema from '../../database/schema/UserSchema';
 import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
-import useProfileHook from '../core/profile/useProfileHook';
+import useUserAuthHook from '../core/auth/useUserAuthHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been changed from `useProfileHook` to `useUserAuthHook`. Ensure that the new hook `useUserAuthHook` provides all the necessary functionalities required by this module. If it doesn't, you might need to refactor the code to accommodate the changes or consider extending the new hook to include missing functionalities.
```diff
- import useProfileHook from '../core/profile/useProfileHook';
+ import useUserAuthHook from '../core/auth/useUserAuthHook';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->